TelosLabs/giving-connection
# Feature/color blocks in edit nonprofit

### Context
Edit my nonprofit required blocks to show a more specific separation between fields. 

### What changed
- Icons
- Blue Text with specific font weight
- Population Served changed to Populations Served
-  Consisten Separation between fields

### How to test it

1. `hivemind s`
2. Log in
3. Edit any of your nonprofits
4. Check design

### References

[Notion ticket](https://www.notion.so/668500b1d20944abb2fec3df69cfcb96?v=4f86b54e86e64ac9a92881430281e753&p=29ec4deb6f4246b89b286486a26adead&pm=s)
